schema/page.json

```diff
@@ -14,13 +14,13 @@
             },
             {
                 "question_id" : "Unique-ID",
-                "reponse_format" : "multipleChoice",
+                "reponse_format" : "mutipleChoice",
                 "question":"Did you enjoy the course?",
                 "options":["Not at all","It was an average course","It was an excellent course"],
             },
             {
                 "question_id" : "Unique-ID",
-                "reponse_format" : "freeResponse",
+                "reponse_format" : "freeReponse",
```

Comment:
Schema.json is non-critical I believe, but you're right about the typo. That should still be addressed.